BraintreeCore/src/main/java/com/braintreepayments/api/UserCanceledException.java

```diff
@@ -5,7 +5,28 @@
  */
 public class UserCanceledException extends BraintreeException {
 
+    private boolean isExplicitCancellation;
```

Comment:
```suggestion
    private boolean isExplicitCancelation;
```

Reply:
I know this one requires using the refactor tool, but we should use the single `l` here for consistency.

Reply:
Good call - will update 👍

Reply:
We made this change but Android Studio is marking it as a typo. Both spellings are technically correct, but according to the internet "cancellation" is far more common. This is the first instance of the word "cancellation" in our code so we have an opportunity to define the style. In my opinion the Android Studio typo underline is distracting and makes our code appear less clean. What do others think?

Reply:
We decided on one `l` a while back actually. I think there's a BT style guide somewhere that mentions this spelling as preferred. We use this spelling in a number of places and it would be a breaking change if we went the two `ll` route.